src/Lesson2.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Date;
+
+public class Lesson2
+{
+    public static void main(String[] args)
+    {
+        boolean result = checkSum(10,10);
+        System.out.print(result + "\n");
+
+        checkValue(10);
+
+        checkNegative(0);
+        System.out.print(result + "\n");
+
+        printN("Привет!", 5);
+
+        Date dt = new Date();
+        int year = dt.getYear() + 1900;
+        result = ifLeapYear(year);
+        System.out.print(result + "\n");
+    }
+
+    public static boolean checkSum (int a, int b)
+    // проверка суммы чисел на принадлежность диапазону [10; 20]
+    {
+        int summa = a + b;
```

Comment:
Транслит не приветствуется
```suggestion
        int sum = a + b;
```